service/history/deletemanager/delete_manager.go

```diff
@@ -30,7 +30,6 @@ import (
 	"context"
 
 	commonpb "go.temporal.io/api/common/v1"
-
```

Comment:
nit: preserve the empty line here. It separates server repo imports from others.

Reply:
Done